reana_client/cli/ping.py

```diff
@@ -81,3 +81,38 @@ def version(ctx):  # noqa: D301
     \t $ reana-client version
     """
     click.echo(__version__)
+
+
+@configuration_group.command("workspaces")
+@click.pass_context
+@add_access_token_options
+@check_connection
+def workspaces(ctx, access_token):  # noqa: D301
+    """List of available workspaces.
+
+    The `workspaces` command lists all the available workspaces.
+    It also returns the default workspace defined by the admin.
+
+    Examples: \n
+    \t $ reana-client workspaces
+    """
+    try:
+        from reana_client.api.client import workspaces
+
+        response = workspaces(access_token)
+        click.echo(
+            click.style(
+                "List of available workspaces: {0}\n"
+                "Default workspace: {1}".format(
+                    ",".join(response.get("workspaces_available")),
```

Comment:
Minor: empty space could be added after the comma. Here is what I get if there are multiple workspace paths available:
```
reana ❯ reana-client workspaces
List of available workspaces: /tmp/reana,/var/reana
Default workspace: /var/reana
```